src/OrchardCore/OrchardCore.DisplayManagement/TagHelpers/DateTimeTagHelper.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace OrchardCore.DisplayManagement.TagHelpers
+{
+    [HtmlTargetElement("datetime")]
```

Comment:
@piedone This will render as `<datetime />` which is not a valid HTML tag. It suggests that it will be rendered by as a javascript web component that is parsed by Vue or Angular for example. This PR is incomplete for that matter.

Reply:
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time
I would rather use a valid HTML tag that can be mutated by a javascript component if required.